addons/project_mrp/models/project.py

```diff
@@ -13,12 +13,57 @@ class Project(models.Model):
 
     def action_view_mrp_production(self):
         self.ensure_one()
-        return self.analytic_account_id.action_view_mrp_production()
+        action = self.analytic_account_id.action_view_mrp_production()
+        if self.production_count > 1:
+            action["view_mode"] = 'tree,form,kanban,calendar,pivot,graph'
+        return action
 
     def action_view_mrp_bom(self):
         self.ensure_one()
-        return self.analytic_account_id.action_view_mrp_bom()
+        action = self.analytic_account_id.action_view_mrp_bom()
+        if self.bom_count > 1:
+            action['view_mode'] = 'tree,form,kanban'
+        return action
 
     def action_view_workorder(self):
         self.ensure_one()
-        return self.analytic_account_id.action_view_workorder()
+        action = self.analytic_account_id.action_view_workorder()
+        if self.workorder_count > 1:
+            action['view_mode'] = 'tree,form,kanban,calendar,pivot,graph,gantt'
+        return action
+
+    # ----------------------------
+    #  Project Updates
+    # ----------------------------
+
+    def _get_stat_buttons(self):
+        buttons = super(Project, self)._get_stat_buttons()
+        if self.user_has_groups('mrp.group_mrp_user'):
+            buttons.extend([{
+                'icon': 'wrench',
+                'text': _('Manufacturing Orders'),
+                'number': self.production_count,
+                'action_type': 'object',
+                'action': 'action_view_mrp_production',
+                'show': self.user_has_groups('mrp.group_mrp_user') and self.production_count > 0,
```

Comment:
Since we are sure the user has the group, you cna directly do this:
```suggestion
                'show': self.production_count > 0,
```